generators/chipyard/src/main/scala/IOBinders.scala

```diff
@@ -384,3 +385,43 @@ class WithDontTouchPorts extends OverrideIOBinder({
   (system: DontTouch) => system.dontTouchPorts(); (Nil, Nil)
 })
 
+class ClockWithFreq(val freqMHz: Double) extends Bundle {
+  val clock = Clock()
+}
+
+class WithDividerOnlyClockGenerator extends OverrideLazyIOBinder({
+  (system: HasChipyardPRCI) => {
+    // First connect the implicit clock
```

Comment:
Does order matter here? What happens if you connect the implicit clock after the referenceClocks + "PLL"

Reply:
No, there should be another comment here, the two things this code does is
1. Connect the implicit clock
2. Connect explicitly requested clocks